Client/Auth/index.ts

```diff
@@ -58,8 +58,20 @@ export class Auth {
 		this.connection.token = value
 	}
 
+	setTempToken(value: string | undefined) {
+		if (!value)
+			window.sessionStorage.removeItem("authData")
```

Comment:
why? doesnt seem appropriate to do this on this method, setTempToken logs you out if its null?